README.md

```diff
@@ -76,170 +48,92 @@ module.exports = {
   context: __dirname,
   entry: './assets/js/index',
   output: {
-      path: path.resolve('./assets/webpack_bundles/'),
-      filename: "[name]-[hash].js"
+    path: path.resolve('./assets/webpack_bundles/'),
+    filename: "[name]-[hash].js"
   },
-
   plugins: [
     new BundleTracker({filename: './webpack-stats.json'})
-  ]
+  ],
 }
 ```
 
+The configuration above expects the `index.js` (the app entrypoint file) to live inside the `/assets/js/` directory (this guide going forward will assume that all front-end related files are placed inside the `/assets/` directory, with the different kinds of files arranged within its subdirectories).
 
-<br>
+The generated compiled files will be placed inside the `/assets/webpack_bundles/` directory and the file with the informations regarding the bundles and assets (`webpack-stats.json`) will be stored in the project root.
 
-### Default Configuration
-```python
-WEBPACK_LOADER = {
-    'DEFAULT': {
-        'CACHE': not DEBUG,
-        'BUNDLE_DIR_NAME': 'webpack_bundles/', # must end with slash
-        'STATS_FILE': os.path.join(BASE_DIR, 'webpack-stats.json'),
-        'POLL_INTERVAL': 0.1,
-        'TIMEOUT': None,
-        'IGNORE': [r'.+\.hot-update.js', r'.+\.map'],
-        'LOADER_CLASS': 'webpack_loader.loader.WebpackLoader',
-    }
-}
-```
-
-<br>
+### Compiling the front-end assets
 
-#### CACHE
-```python
-WEBPACK_LOADER = {
-    'DEFAULT': {
-        'CACHE': not DEBUG
-    }
-}
-```
-When `CACHE` is set to True, webpack-loader will read the stats file only once and cache the result. This means web workers need to be restarted in order to pick up any changes made to the stats files.
+Generally, `webpack-bundle-tracker` is used as part of an already existing build pipeline that is provided by a front-end framework/library, such as React.
 
-<br>
+However, if you wish to compile the files by yourself, you can do so by running `./node_modules/.bin/webpack --config webpack.config.js --watch`. We do so on our examples, which you can check out by going [here](https://github.com/django-webpack/django-webpack-loader/tree/master/examples).
```

Comment:
`npx /webpack --config webpack.config.js --watch` is probably the modern way to run it.